### PR TITLE
[webview_flutter] Remove un-used method params

### PR DESCRIPTION
**Pull request by iskakaushik**

### CODE REVIEW

File: packages/webview_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.3.6+1
+
+* Remove un-used method params in webview\_flutter
```
Comment (mklim) [inline packages/webview_flutter/CHANGELOG.md]: nit:

```suggestion
* Remove un-used method params in webview_flutter
```

Comment (cyanglaz) [inline packages/webview_flutter/CHANGELOG.md]:
```suggestion
* Remove un-used method params.
```

Comment (iskakaushik) [reply packages/webview_flutter/CHANGELOG.md]: in markdown '_' needs to be escaped, or it waits for it to be closed as an underline.

Comment (mklim) [reply packages/webview_flutter/CHANGELOG.md]: Oh, right. My mistake!

**Pull request merged**